.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,347 @@
+  subtract 4 columns for the indent on the left and 4 more for symmetry on the
+  right, we’re left with 72 columns.
+- git format-patch --stdout converts a series of commits to a series of emails,
+  using the messages for the message body.  Good email netiquette dictates we
+  wrap our plain text emails such that there’s room for a few levels of nested
+  reply indicators without overflow in an 80 column terminal.
+
+<a name="CodeApproval" />
+
+### 5. Code Approval Process
+
+This section describes the code approval process that is used for code
+contributions.  This is how to get your changes into bcoin.
+
+<a name="CodeReview" />
+
+### 5.1 Code Review
+
+All code which is submitted will need to be reviewed before inclusion into the
+master branch.  This process is performed by the project maintainers and usually
+other committers who are interested in the area you are working in as well.
+
+##### Code Review Timeframe
+
+The timeframe for a code review will vary greatly depending on factors such as
+the number of other pull requests which need to be reviewed, the size and
+complexity of the contribution, how well you followed the guidelines presented
+on this page, and how easy it is for the reviewers to digest your commits.  For
+example, if you make one monolithic commit that makes sweeping changes to things
+in multiple subsystems, it will obviously take much longer to review.  You will
+also likely be asked to split the commit into several smaller, and hence more
+manageable, commits.
+
+Keeping the above in mind, most small changes will be reviewed within a few
+days, while large or far reaching changes may take weeks.  This is a good reason
+to stick with the [Share Early, Share Often](#ShareOften) development practice
+outlined above.
+
+##### What is the review looking for?
+
+The review is mainly ensuring the code follows the [Development
+Practices](#DevelopmentPractices) and [Code Contribution
+Standards](#Standards).  However, there are a few other checks which are
+generally performed as follows:
+
+- The code is stable and has no stability or security concerns
+- The code is properly using existing APIs and generally fits well into the
+  overall architecture
+- The change is not something which is deemed inappropriate by community
+  consensus
+
+<a name="CodeRework" />
+
+### 5.2 Rework Code (if needed)
+
+After the code review, the change will be accepted immediately if no issues are
+found.  If there are any concerns or questions, you will be provided with
+feedback along with the next steps needed to get your contribution merged with
+master.  In certain cases the code reviewer(s) or interested committers may help
+you rework the code, but generally you will simply be given feedback for you to
+make the necessary changes.
+
+This process will continue until the code is finally accepted.
+
+<a name="CodeAcceptance" />
+
+### 5.3 Acceptance
+
+Once your code is accepted, it will be integrated with the master branch.
```

Comment:
>Once your code is accepted, a maintainer will merge it with the master branch. Typically it will be rebased and fast-forward merged to master as we prefer to keep a clean commit history over a tangled weave of merge commits.  However, regardless of the specific merge method used, the code will be integrated with the master branch and the pull request will be closed.
>
>Rejoice! You are now a `bcoin` [contributor](https://github.com/bcoin-org/bcoin/graphs/contributors)!